content/en/blog/_posts/2023-12-14-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Cloud Provider Integrations Are Now Separate Components"
+date: 2023-12-14T09:30:00-08:00
+slug: cloud-provider-integration-changes
+---
+
+**Authors:** Michael McCune (Red Hat), Andrew Sy Kim (Google)
+
+For Kubernetes v1.29, you need to use additional components to integrate your
+Kubernetes cluster with a cloud infrastructure provider. By default, Kubernetes
+v1.29 disallows running the legacy compiled-in cloud provider integrations.
```

Comment:
Alternative suggestion for this sentence, feel free to ignore if it's not as succinct:
```
By default, Kubernetes 1.29 disallows components configured with legacy compiled-in cloud provider integrations
```

Reply:
thanks Andrew, i'll update after lunch